src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackageArgs.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using NuGet.Common;
+using NuGet.Configuration;
+using NuGet.Packaging.Core;
+using System.Threading;
+
+namespace NuGet.CommandLine.XPlat
+{
+    public class ListPackageArgs
+    {
+        public ILogger Logger { get; }
+        public string Path { get; set; }
+        public List<PackageSource> PackageSources { get; set; }
+        public bool Framework { get; set; }
+        public IList<string> Frameworks { get; set; }
```

Comment:
why 2 different fields for framework? you can easily achieve the same thing with single parameter...

Also, why have a list when we only support taking single framework?

Reply:
We support taking multiple frameworks if the are separated with commas. The boolean is unnecessary. I'll get rid of it